src/MessagingBubble/NormalMessagingBubble.tsx

```diff
@@ -99,13 +107,96 @@ export const NormalMessagingBubble: React.FC<Props> = ({
         </div>
       </FlexBox>
       <FlexBox className={timeAndBubbleContainerClasses}>
-        {hideBubble && (
-          <span className={timestampClassNames}>{_formatDateForTimestamp(timestamp)}</span>
-        )}
+        {_renderMetadata({
+          renderHere: hideBubble,
+          timestamp,
+          onClickDeleteButton,
+          metadataClassNames,
+          actionButtonClassNames,
+        })}
         {attachments?.length > 0 && (
           <FlexBox className={attachmentClassNames}>{attachments}</FlexBox>
         )}
       </FlexBox>
     </FlexBox>
   );
 };
+
+// Helper function: Format a Date for our pretty timestamps.
+//  Always returns "xx:xx <AM/PM>" format.
+function _formatDateForTimestamp(date: Date): string {
+  return moment(date).format("h:mm A");
+}
+
+// Helper function: renders metadata, consisting of either the action button
+//  or a non-interactable timestamp
+function _renderMetadata({
+  renderHere,
```

Comment:
hmm what will this be used for in practice? it seems like instead there could be something like

``` 
{hideBubble && _renderMetadata({ 
    ...
    })
}
```

Reply:
i think they're equivalent, but maybe the prop isn't necessary inside of the ?

Reply:
I think you're right! 👍🏻